wfl/fit/error.py

```diff
@@ -375,9 +378,56 @@ def _annotate_parity_plot(ax, property, ref_property_prefix, calc_property_prefi
 
 def _annotate_error_plot(ax, property, ref_property_prefix, calc_property_prefix):
     ax.set_title(f"{property} error")
-    ax.set_xlabel(f"{ref_property_prefix}{property}, {_units(property, 'parity')}")
-    ax.set_ylabel(f"{calc_property_prefix}{property} error, {_units(property, 'error')}") 
+    ax.set_xlabel(f"{ref_property_prefix}{property}, {select_units(property, 'parity')}")
+    ax.set_ylabel(f"{calc_property_prefix}{property} error, {select_units(property, 'error')}") 
     ax.grid(color='lightgrey', ls=':')
     ax.set_yscale('log')
 
 
+def errors_dumps(errors, precision=2, file=sys.stdout):
+    """converts errors dictionary to dataframe and prints nicely.
+    
+    Parameters
+    ----------
+    errors: dict
+        Dictionary returned by wfl.fit.error.calculate()
+    precision: int, default 2
+        Float precision when printing
+    file: obj, default sys.stdout
+        Object to write the table to. 
+    """
+
+    df = errors_to_dataframe(errors) 
+    with pd.option_context('display.max_rows', None,
+                           'display.max_columns', None,
+                           'display.precision', precision,
+                           'display.colheader_justify', 'center'):
+        print(df, file=sys.stdout)
```

Comment:
By analogy to things like `json.dumps()`, this should return a string, not actually print. Why not use `DataFrame.to_string()`?

Reply:
I see what you meant, please resolve this if the update looks ok.